python/pyproject.toml

```diff
@@ -32,3 +32,8 @@ py-modules = ["slice2py"]
 
 [project.scripts]
 slice2py = "slice2py:main"
+
+[tool.pyright]
+include = ["test"]
+typeCheckingMode = "standard"
+extraPaths = ["python", "../scripts"]
```

Comment:
Why do we need `scripts` here? This is the pyproject for build Ice for Python, and `../scripts` is unrelated python code.